### DISCUSSION TITLE
On chart/list add the option to find charts that are not used in dashboards

### DISCUSSION THREAD

**Discussion by SharonCastel** [apache/superset#26605] [Ideas]
On the list of charts, we would like to have the list of the charts that are not attached to dashboard in order to clean the environment and delete the charts that are not in use.
It would be great to have this option here:
![image](https://github.com/apache/superset/assets/93184294/93ec96aa-b073-41c1-9026-0d9a16aefcc2)

**Comment by LyleScott** [comment]
This would be so helpful for cleaning stuff up.

**Comment by aayush-kosh** [comment]
Are there any plans to implement this?

**Comment by mtrentz** [comment]
I would also love a solution for this.

Although this doesn't actually make it easier to delete charts, this query here shows you all dangling ones not attached to any dashboard.

```
SELECT s.id, s.slice_name, 
  CONCAT('<a href="https://{YOUR_DOMAIN.com}/explore/?slice_id=', s.id, '">Link</a>') as link
FROM slices s
LEFT JOIN dashboard_slices ds ON s.id = ds.slice_id
WHERE ds.slice_id IS NULL;
```
